tldr/cross_file_calls.py

```
@@ -1557,6 +1567,75 @@ def _parse_elixir_import_node(node, source: bytes) -> dict | None:
 
 
+def parse_zig_imports(file_path: str | Path) -> list[dict]:
+    if not TREE_SITTER_ZIG_AVAILABLE:
+        return []
+
+    file_path = Path(file_path)
+    try:
+        source = file_path.read_bytes()
+        parser = _get_zig_parser()
+        tree = parser.parse(source)
+    except (FileNotFoundError, Exception):
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)